tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -24,7 +25,11 @@ public TesAdapterController(TesTaskService tesTaskService) {
     }
 
     @GetMapping("/v1/tasks")
-    public ResponseEntity<TesListTasksResponse> listTesTasks() {
-        return new ResponseEntity<>(tesTaskService.listTesTask(), HttpStatus.OK);
+    public ResponseEntity<TesListTasksResponse> listTesTasks(@RequestParam("name_prefix") String name_prefix,
```

Comment:
Please, user camel case for argument names -> `name_prefix` to `namePrefix`, param name in `@RequestParam` shall be left as is.
Since all these parameters are optional, we need to specify it in the annotation like `@RequestParam(name = "name_prefix", required = false)`.